src/WordPress/Hooks.php

```diff
@@ -126,7 +126,7 @@ public function purgeCacheEverything()
         }
     }
 
-    public function purgeCacheByRelevantURLs($postId)
+    public function purgeCacheByRelevantURLs($postId, ...$args)
```

Comment:
@Inwerpsel i'm fresh to this codebase so excuse the potentially stupid question but are you able to clarify for me why we're updating the function arguments? from what i can see, i think we can do it within the method itself without modifying this and causing a potentially breaking change for the plugin.

Reply:
this the explanation from the PR description:
```
Add ...$args to the hook to avoid doing additional queries. This could
likely be used elsewhere in the function instead of querying those
things.
```
I prefer to play safe with a potentially breaking change.

Reply:
To be fair I think it is ok change. @Inwerpsel  is filtering out the case when post was updated and  state wasn't changed (private -> publish). We only interested in (publish -> private). For that we need 3rd parameter https://developer.wordpress.org/reference/hooks/post_updated/

Reply:
@jacobbednarz Updating the function arguments should by itself be safe for the actions that are listed, since the code is already written to support running on multiple actions, each of which can have a different number of arguments.

But maybe it does have some implications if custom actions are added with `cloudflare_purge_url_actions` :thinking: These could have any function signature, while these checks make some assumptions about the presence of the 3rd argument meaning it's a `post_updated` action.

https://github.com/greenpeace/Cloudflare-WordPress/blob/d7784bf47f79dcf3e4f758df5e7f75c696cfc203/src/WordPress/Hooks.php#L149-L155